src/video/wayland/SDL_waylandvideo.h

```diff
@@ -41,6 +41,11 @@ struct qt_surface_extension;
 struct qt_windowmanager;
 #endif /* SDL_VIDEO_DRIVER_WAYLAND_QT_TOUCH */
 
+#ifdef SDL_VIDEO_DRIVER_WAYLAND_TABLET
+//#include "tablet-unstable-v2-protocol.h"
```

Comment:
I think we can actually take out this whole block? If not...

```suggestion
```

Reply:
no, the SDL_WaylandTabletManager is required as part of SDL_VideoData. Unless you want to to use the proper type (and include the `tablet-unstable-v2-protocol.h` (same thing as SDL_waylandevents_c.h).

Reply:
Understood, so the comment is all that needs to be taken out here (that or a note if the eventual goal is to include the real protocol header).